pkgs/by-name/me/meson/package.nix

```diff
@@ -92,6 +114,8 @@ python3.pkgs.buildPythonApplication rec {
   nativeCheckInputs = [
     ninja
     pkg-config
+  ] ++ lib.optionals python3.isPyPy [
```

Comment:
This instance of python3 should be changed by pythonPackages too.

Reply:
by `PythonPackages`? not `Python3Packages`?

Reply:
Yes, that's what I meant.